.gitignore

```diff
@@ -62,9 +62,5 @@ buck-out/
 android/app/libs
 android/keystores/debug.keystore
 
-# generated by bob
-lib/
```

Comment:
I think this line is actually generated when you do a build of the react native project so its more of an issue with the repo config than updating the path. It will just keep getting added back in.

Reply:
Added, will see how that goes 🤷